src/coverlet.console/Program.cs

```diff
@@ -88,7 +90,8 @@ static int Main(string[] args)
                     logger,
                     serviceProvider.GetService<IInstrumentationHelper>(),
                     fileSystem,
-                    sourceTranslator);
+                    sourceTranslator,
```

Comment:
can we use `serviceProvider.GetRequiredService` everywhere where we don't use variable for more than one place?I see some assignation above, `sourceTranslator` is used in more than one place?

Reply:
Done